2018-01-11 22:34:22|ERROR|App.Comment.AddOne|PDOException: Incorrect integer value: 'undefined' for column 'cu_userid' at row 1 in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(338): NotORM_Result->query('INSERT INTO com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(355): NotORM_Result->insert_multi(Array)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(14): NotORM_Result->insert(Array)
#3 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(52): App\Model\Useful->addOne('useful', 'undefined', '61')
#4 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(72): App\Domain\Comment->addOne('useful', 'undefined', '61')
#5 [internal function]: App\Api\Comment->addOne()
#6 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#7 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#8 {main}
2018-01-11 22:34:26|ERROR|App.Comment.AddOne|PDOException: Incorrect integer value: 'undefined' for column 'cu_userid' at row 1 in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(338): NotORM_Result->query('INSERT INTO com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(355): NotORM_Result->insert_multi(Array)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(14): NotORM_Result->insert(Array)
#3 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(52): App\Model\Useful->addOne('notuseful', 'undefined', '61')
#4 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(72): App\Domain\Comment->addOne('notuseful', 'undefined', '61')
#5 [internal function]: App\Api\Comment->addOne()
#6 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#7 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#8 {main}
2018-01-11 22:34:43|ERROR|App.Comment.AddOne|PDOException: Incorrect integer value: 'undefined' for column 'cu_userid' at row 1 in /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php:261
Stack trace:
#0 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(338): NotORM_Result->query('INSERT INTO com...', Array)
#1 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/notorm/src/NotORM/Result.php(355): NotORM_Result->insert_multi(Array)
#2 /Users/lixuanqi/Github/rate/backend/src/app/Model/Useful.php(14): NotORM_Result->insert(Array)
#3 /Users/lixuanqi/Github/rate/backend/src/app/Domain/Comment.php(52): App\Model\Useful->addOne('useful', 'undefined', '61')
#4 /Users/lixuanqi/Github/rate/backend/src/app/Api/Comment.php(72): App\Domain\Comment->addOne('useful', 'undefined', '61')
#5 [internal function]: App\Api\Comment->addOne()
#6 /Users/lixuanqi/Github/rate/backend/vendor/phalapi/kernal/src/PhalApi.php(49): call_user_func(Array)
#7 /Users/lixuanqi/Github/rate/backend/public/index.php(9): PhalApi\PhalApi->response()
#8 {main}
